modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -233,7 +232,7 @@ CREATE TABLE ClickEvent (
 
 Consider this approach if the topic data is in JSON, or if you can use the Iceberg data in its semi-structured format.
 
-The `value_schema_id_prefix` mode translates to the following table format:
+The `value_schema_id_prefix` and `value_schema_latest` modes translate to the following table format:
```

Comment:
Need confirmation from SME that this is also correct for value_schema_latest

Reply:
correct